remove obsolete patch for exact-image

hvoigt opened the pull request:
The include guard that was added by this patch has already made it
upstream. So it is not needed anymore.

Signed-off-by: Heiko Voigt <hvoigt@hvoigt.net>

DomT4 commented:
Remove all the references to the patch, not just the patch itself :)

hvoigt commented:
Just saw that myself when preparing another formula. Fixed, hopefully :)

DomT4 commented on exact-image.rb:
```diff
@@ -18,24 +15,3 @@ def install
 end
 
 __END__
```
Can also kill this line, then you'll be all good :+1:

hvoigt commented:
Thanks for the tip/patience! As you can probably tell, I am not a ruby specialist... just trying to fix an installation issue for me. Updated. Now the integration test still fails because of a SSL certificate error when trying to checkout from the SVN. Is that ok?

DomT4 commented:
No worries! The syntax can take a little while to grasp. It looks like the SVN repo upstream may be having some issues, the SSL cert gets rejected, but if you drop to HTTP instead of HTTPS you get this:

```
svn: E175013: Unable to connect to a repository at URL 'http://svn.exactcode.de/exact-image/trunk'
svn: E175013: Access to 'http://svn.exactcode.de/exact-image/trunk' forbidden
```

Same thing happens when you use the ` svn:// ` protocol instead. Might be an upstream issue.

hvoigt commented:
It seems like it is a new upstream problem, since the tests yesterday still passed and I did not change the url. While at it, added a commit to removing all the unnecessary require 'formula'. It looks good but I do not know how to test this thoroughly. I hope the testsuite would catch if I made a mistake right?

Will send a mail to the exact-image mailinglist whether this is intended or will be fixed.

DomT4 commented:
Might just be a temporary connection problem upstream. It happens.

> While at it, added a commit to removing all the unnecessary require 'formula'

This was dedication, but I don't know if the maintainers here will accept it. Generally "style-only" changes aren't accepted because users get notified about each formula update when they ` brew update `, so a sudden wad of changes can look overwhelming :smile_cat:. But yes, the bot will point out any errors :+1:.

hvoigt commented:
>> While at it, added a commit to removing all the unnecessary require 'formula'

> This was dedication, but I don't know if the maintainers here will accept it. Generally "style-only" changes aren't accepted because users get notified about each formula update when they brew update, so a sudden wad of changes can look overwhelming :smile_cat:. But yes, the bot will point out any errors :+1:.

Well it was a quick change if you know your editor (took me less than 5 minutes). I thought it would be nice for others, like me, so they do not copy from a source that still has it and you have to tell them to remove it. But I can imagine that a user might be concerned by a change like this. Removed the extra file changes.

hvoigt commented:
Ok I got an answer from the developers. They are using a cacert certificate. Is it a feasible solution to add the cacert root certificate (http://www.cacert.org) to the build bot?

DomT4 commented:
Sadly, probably not. Adding root certificates is a fairly dangerous game, and beyond that, given the way the bots are set up in terms of VM's at the moment, a nightmare to implement. @mikemcquaid is the master of all things bot though; He'll know whether it's a go or not as an idea.

hvoigt commented:
I just got a tip from from the developer that there are actually releases and why we are not using that. So I will open a PR over in the main repository with basically this formulae switched over to the tarball url.

hvoigt commented:
And here is the PR https://github.com/Homebrew/homebrew/pull/38368

DomT4 commented:
Nice! That works, especially if we can avoid using an unrecognised SSL/TLS cert. You can turn this PR into a PR to delete the formula here if you like.

hvoigt commented:
Thanks. Done, hopefully. Just need to wait on the other pull request.
